app/scenes/ChangePassword/index.js

```diff
@@ -51,14 +51,6 @@ class ChangePasswordScreen extends Component {
     const { authToken, user, updateUserDetailsSuccessful, replaceReponses } = this.props;
 
     return updatePassword(authToken, oldPassword, password)
-      .then(() => {
-        user.privateKey = getPrivateKey({
-          userId: user._id,
-          email: user.email,
-          password,
-        });
-        return replaceReponses(user);
-      })
```

Comment:
@Matchdev120  This is neccessary api call for response encryption/decryption.

updating user's password means updating password for encrypting user's previous responses, so they should be re-encrypted.
If this causes speed issue, it should be fixed different way rather than deleting logic.